access-control-app/src/cmr/access_control/services/acl_service.clj

```diff
@@ -31,18 +30,11 @@
    [cmr.transmit.urs :as urs]
    [cmr.umm-spec.acl-matchers :as acl-matchers]))
 
-(defn- context->user-id
```

Comment:
Can you double check that this func is not being used anywhere before you delete?

I see in group_service.clj line 65 that some func similarly named is being used there